levels/lovelace_tower/objectives/api-01-local-function/walkthrough.md

```diff
@@ -0,0 +1,4 @@
+# Helpful Resources
+
+- [MDN - String replaceAll method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll)
```

Comment:
Not sure what (if anything) more we wanted to do with the walkthrough for this objective. We're not really introducing anything new to the player, besides maybe the `replaceAll` method. Welcome to any ideas!

Reply:
Maybe suggest they revisit the Turing initiation in part 1? That covers the local API topic

Reply:
Is there a specific part of the Turing initiation you're referring to? I was having trouble finding this while playing through the scenario. There is the tomb at the beginning of the `lovelace_tower/default` map that links to resources for APIs, if that's what you are pointing to!

Reply:
Tomb -> tome